deploy/clientcomm.tf

```diff
@@ -226,6 +226,20 @@ resource "aws_route_table_association" "clientcomm" {
   route_table_id = "${aws_route_table.clientcomm.id}"
 }
 
+// TODO: provision the DNS zone here instead of looking it up?
+data "aws_route53_zone" "clientcomm" {
+  // transform 'https://multnomah.clientcomm.org' -> 'clientcomm.org.'
+  name = "${replace(replace(var.deploy_base_url, "/https:\\/\\//", ""), "/^[^\\.]+\\./", "")}."
```

Comment:
Yeah... I wish we could use JS or anything in the terraform file, but we're limited to the [terraform interpolation syntax](https://www.terraform.io/docs/configuration/interpolation.html).

Fortunately, it's pretty easy to audit that changes don't break because the `terraform plan` command will show you everything terraform plans before you execute it.